src/libraries/System.Runtime/tests/System/Reflection/PointerTests.cs

```diff
@@ -220,6 +222,25 @@ public void IntPtrMethodParameter(int value)
             method.Invoke(obj, new object[] { (IntPtr)value, value });
         }
 
+        public static IEnumerable<object[]> PointersUInt =>
+            new[]
+            {
+                new object[] { UIntPtr.Zero },
+                new object[] { 0 },
+                new object[] { 1 },
+                new object[] { uint.MinValue },
+                new object[] { uint.MaxValue },
+            };
+
+        [Theory]
+        [MemberData(nameof(Pointers))]
```

Comment:
Is this supposed to be `PointersUInt`?

Reply:
If not, I don't see use of `PointersUInt` anywhere.

Reply:
Right